pocs/dome/astrohaven.py

```diff
@@ -157,6 +191,11 @@ def _full_move(self, send, target_feedback, feedback_countdown=1):
             target_feedback: The response code to compare to the response from the dome;
                 this is a string of one ASCII character. See Protocol above for the codes;
                 while the dome is moving, it echoes the command code sent.
+            feedback_countdown: Number of times the target_feedback needs to be received
```

Comment:
Seems like most of this would be better placed in the `close` method rather than in a private method. But maybe you wanted to bury the details...